### PR TITLE
IGAPP-1087: Fix PDF on Android

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: native/src/routes/PDFViewModal.tsx [hunk 1]
```diff
@@ -38,6 +38,7 @@ const PDFViewModal = ({ route, navigation: _navigation }: PropsType): ReactEleme
         source={{
           uri: url,
         }}
+        trustAllCerts={false}
```
Comment (steffenkleinle) [inline native/src/routes/PDFViewModal.tsx:41]: Could you please explain this? Why does this help?

File: release-notes/unreleased/IGAPP-1087-fix-pdf-on-android.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+issue_key: IGAPP-1087
+show_in_stores: true
+platforms:
+  - android
```
Comment (steffenkleinle) [inline release-notes/unreleased/IGAPP-1087-fix-pdf-on-android.yml:4]: Didn't happen on ios?

**Review by steffenkleinle — APPROVED** [review]
Tested on android.

File: release-notes/unreleased/IGAPP-1087-fix-pdf-on-android.yml (hunk 2 above)
Comment (SteffiStuffel) [reply release-notes/unreleased/IGAPP-1087-fix-pdf-on-android.yml:4]: No only an android issue

File: native/src/routes/PDFViewModal.tsx (hunk 1 above)
Comment (SteffiStuffel) [reply native/src/routes/PDFViewModal.tsx:41]: Some bug in react-native-pdf or react-native-fetch-blob. Should be default be false, but does not work correctly. See here:
https://github.com/wonday/react-native-pdf/issues/460

Comment (steffenkleinle) [reply native/src/routes/PDFViewModal.tsx:41]: Thanks! According to the docs the default is true.

**Pull request merged**